lib/core/hooks.js

```diff
@@ -40,7 +40,7 @@ export async function buildHook (moduleContainer, options, logger) {
     // Determine "config.release" automatically from local repo if not provided.
     try {
       // @ts-ignore
-      const SentryCli = (await import('@sentry/cli')).default
+      const SentryCli = await import('@sentry/cli')
```

Comment:
Try something like this instead:
```suggestion
      const SentryCli = await (import('@sentry/cli').then(m => m.default || m))
```
Whether the import contains the `default` export can depend on the user setting so the code should handle both cases.

Reply:
Thanks! that's a better option